import scala.scalanative.unsafe._

import stdio._

object Hello {
  def main(args: Array[String]): Unit = {
    throw new Exception()
  }
}
